src/main/groovy/br/com/bluesoft/bee/database/reader/PostgresDatabaseReader.groovy

```diff
@@ -227,8 +229,8 @@ class PostgresDatabaseReader implements DatabaseReader {
 	'''
 
     final static def INDEXES_QUERY_9_6 = '''
-		select ct.relname as table_name, ci.relname as index_name, i.indisunique as uniqueness, am.amname as index_type, 
-		      pg_get_indexdef(ci.oid, (i.keys).n, false) as column_name, 
+		select n.nspname schemaname,  ct.relname as table_name, ci.relname as index_name, i.indisunique as uniqueness, 
```

Comment:
the variable name seems to indicate that it is valid only for postgres 9.6... but, since you changed it, i believe it is valid for every version newer than that...
i think you also changed the version getter to retrieve only the major version (9, in this case)...
don't you think that would be better to rename the variable (so that it becomes clearer when this query will be used)?

--

o nome da variável indica que é válido apenas para postgres 9.6... mas como vc mudou, creio que seja para qualquer (talvez qualquer uma superior a 9)...
vc tbm mudou pelo que entendi pra retornar apenas a versão major (9, nesse caso)
se é isso mesmo, talvez seja melhor renomear a variável para ficar mais claro?

Reply:
maybe something like INDEXES_QUERY_FOR_VERSIONS_9_OR_ABOVE...